### PR TITLE
tickets/DM-30379

### PR DESCRIPTION
**Pull request by mareuter**
Update DSM version.

### CODE REVIEW

File: cycle/cycle.env [hunk 1]
```diff
@@ -77,7 +77,7 @@ ts_watcher=1.5.3
 ts_scheduler=1.5.3
 ts_mtdome=1.1.0
 ts_MTDomeTrajectory=0.6.0
-ts_dsm=2.2.1
+ts_dsm=2.2.2
```
Comment (tribeiro) [inline cycle/cycle.env:80]: revision 2 is already closed. Do you want to update this to revision 3 and merge before I close the revision?

Comment (mareuter) [reply cycle/cycle.env:80]: I was assuming this would go on version 3.

Comment (tribeiro) [reply cycle/cycle.env:80]: yes.. so you need to update the rev number at the top... if you rebase the develop now you will get it.

Comment (mareuter) [reply cycle/cycle.env:80]: Oh, OK. Missed that. Will fix it.

Comment (mareuter) [reply cycle/cycle.env:80]: Ugh. Rebased the wrong thing. I'm going to blow this branch away and start over.

**Pull request closed without merging**